apps/dash-image-annotation/app.py

```diff
@@ -182,8 +190,60 @@ def shape_data_remove_timestamp(shape):
         html.Div(
             id="banner",
             children=[
-                html.H1("Bounding Box Classification App", id="title"),
-                html.Img(id="logo", src=app.get_asset_url("dash-logo-new.png")),
+                dbc.Row(
+                    [
+                        dbc.Col(
+                            html.Img(
+                                id="logo", src=app.get_asset_url("dash-logo-new.png")
+                            ),
+                            width=2,
+                            # align="center",
+                        ),
+                        dbc.Col(
+                            html.H1("Bounding Box Classification App", id="title"),
+                            width=4,
+                            align="center",
+                        ),
+                        dbc.Col(
+                            [
+                                dbc.Button(
+                                    "Readme",
```

Comment:
What would you think of using "about" or "learn more" instead of "readme"? I think "about" has my preference but it's not a strong one.

Reply:
Yes, both are better. I'm leaning a bit more to "learn more" as it seems more explicit to me.